pkg/library/flatten/testdata/workspace-env/adds-workspace-env-written-as-json.yaml

```diff
@@ -0,0 +1,84 @@
+name: "Adds JSON workspace environment variables from plugin "
```

Comment:
File name and case name does not seem to be really the same.

Reply:
Copy-paste error :)